examples/src/bin/ech-client.rs

```diff
@@ -28,7 +28,7 @@
 //! "SSL_ECH_STATUS": "success"
 //! ```
 
-use std::error::Error;
+use core::error::Error;
```

Comment:
I think it would be better to update like this in general:

```diff
-use core::error::Error;
+use core::error::Error as StdError;
```

This kind of update would increase the divergence from upstream RUSTLS. I think this kind of update should be done in upstream RUSTLS.